docs/user-guide-stack.md

```diff
@@ -35,15 +35,26 @@ packages.  The overlay specifies which `extra-deps` (here: clock-0.1.1)
 we wanted to overlay over the stackage snapshot, and what local
 packages we want (here: my-package).
 
-We will then create the following `nix/pkgs.nix` file:
+We will then create the following `nix/default.nix` file:
 
 ```nix
-let stack-pkgs = import ./.stack-pkgs.nix; in
-{ stackage, ... }:
-{ pkg-def = stackage.${stack-pkgs.resolver};
-  inherit (stack-pkgs) overlay;
-}
+{ pkgs ? import <nixpkgs> {} }:
+
+let
+  haskell = import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz) { inherit pkgs; };
```

Comment:
doesn't this contradict with `overrideWith` described in basic user guide?

Reply:
Yes, actually we need update `cabal-to-nix` code generation to be more similar to `stack-to-nix`, and delete those big chunks of boilerplate code in the *Setup* section.

Reply:
And the proper way to override is with `hackageSourceJSON` and `stackageSourceJSON`? If so shouldn't they appear here?

Reply:
There should be a section in the user guide for pinning and overriding pins. But for now, #77.